products/pubsub/terraform.yaml

```diff
@@ -16,6 +16,9 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   Topic: !ruby/object:Overrides::Terraform::ResourceOverride
     id_format: "projects/{{project}}/topics/{{name}}"
     error_retry_predicates: ["pubsubTopicProjectNotReady"]
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
```

Comment:
does this merge on top of the api value? I assume it does (otherwise exclude would be true) so we could just have the parent_resource_attribute

Reply:
For some reason it wasn't merging that field. Possibly because of a default value?